### DISCUSSION TITLE
MAC Sonoma, scrolling stopped working

### DISCUSSION THREAD

**Discussion by kubebn** [linearmouse/linearmouse#624] [General]
Hi, the issue did not occur immediately after the upgrade, but I have now noticed that the scrolling function has stopped working on my mouse (Logitech Superlight). Has anyone else experienced the same issue?

**Comment by johnjdavisiv** [comment]
Same experience for me but with the acceleration - my scrolling still works as expected, but my pointer acceleration is back!
